content/api/events.apib

```diff
@@ -0,0 +1,179 @@
+FORMAT: 1A
```

Comment:
validator for string length of wildcard domain for perf improvements of wildcard (enforce minimum number of characters x, where x is the minimum for a domain (which is 2), so need to determine if this number is performant or not, and re-evaluate if not